translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"\n"
+"\t\t# Listen on port 8888 locally, forwarding to 5000 in the pod\n"
+"\t\tkubectl port-forward mypod 8888:5000\n"
+"\n"
+"\t\t# Listen on a random port locally, forwarding to 5000 in the pod\n"
+"\t\tkubectl port-forward mypod :5000\n"
+"\n"
+"\t\t# Listen on a random port locally, forwarding to 5000 in the pod\n"
+"\t\tkubectl port-forward mypod 0:5000"
+msgstr ""
+"\n"
+"\t\t# Escuta nas portas locais 5000 e 6000, e redireciona os dados de/para "
+"as portas 5000 e 6000 no pod\n"
+"\t\tkubectl port-forward mypod 5000 6000\n"
+"\n"
+"\t\t# Escuta na porta local 8888 locally, e redireciona para a porta 5000 no "
+"pod\n"
+"\t\tkubectl port-forward mypod 8888:5000\n"
+"\n"
+"\t\t# Escuta uma porta local aleatória, e redireciona para a porta 5000 no "
+"pod\n"
+"\t\tkubectl port-forward mypod :5000\n"
+"\n"
+"\t\t# Escuta uma porta local aleatória, e redireciona para a porta 5000 no "
+"pod\\n\n"
+"\t\tkubectl port-forward mypod 0:5000"
+
+#: pkg/kubectl/cmd/drain.go:118
+msgid ""
+"\n"
+"\t\t# Mark node \"foo\" as schedulable.\n"
+"\t\t$ kubectl uncordon foo"
+msgstr ""
+"\n"
+"\t\t# Marca o node “foo” como agendável.\n"
+"\t\t$ kubectl uncordon foo"
+
+#: pkg/kubectl/cmd/drain.go:93
+msgid ""
+"\n"
+"\t\t# Mark node \"foo\" as unschedulable.\n"
+"\t\tkubectl cordon foo"
+msgstr ""
+"\n"
+"\t\t# Marca o node “foo” como não atenuável.\n"
+"\t\tkubectl cordon foo"
+
+#: pkg/kubectl/cmd/patch.go:66
+msgid ""
+"\n"
+"\t\t# Partially update a node using strategic merge patch\n"
+"\t\tkubectl patch node k8s-node-1 -p '{\"spec\":{\"unschedulable\":true}}'\n"
+"\n"
+"\t\t# Partially update a node identified by the type and name specified in "
+"\"node.json\" using strategic merge patch\n"
+"\t\tkubectl patch -f node.json -p '{\"spec\":{\"unschedulable\":true}}'\n"
+"\n"
+"\t\t# Update a container's image; spec.containers[*].name is required "
+"because it's a merge key\n"
+"\t\tkubectl patch pod valid-pod -p '{\"spec\":{\"containers\":[{\"name\":"
+"\"kubernetes-serve-hostname\",\"image\":\"new image\"}]}}'\n"
+"\n"
+"\t\t# Update a container's image using a json patch with positional arrays\n"
+"\t\tkubectl patch pod valid-pod --type='json' -p='[{\"op\": \"replace\", "
+"\"path\": \"/spec/containers/0/image\", \"value\":\"new image\"}]'"
+msgstr ""
+"\n"
+"\t\t# Atualiza parcialmente um node utilizando a estratégia merge patch\n"
+"\t\tkubectl patch node k8s-node-1 -p ‘{“spec”:{“unschedulable”:true}}’\n"
+"\n"
+"\t\t# Atualiza parcialmente um node identificado pelo tipo e nome no arquivo "
+"“node.json” utilizando a estratégia merge patch\n"
+"\t\tkubectl patch -f node.json -p ‘{“spec”:{“unschedulable”:true}}’\n"
+"\n"
+"\t\t# Atualiza uma image um container; spec.containers[*].name é requerido "
+"porque é a chance de merge\n"
+"\t\tkubectl patch pod valid-pod -p ‘{“spec”:{“containers”:"
+"[{“name”:”kubernetes-serve-hostname”,”image”:”new image”}]}}’\n"
+"\n"
+"\t\t# Atualiza uma image um container utilizando o json patch com positional "
```

Comment:
missed the word image here.
"Atualiza uma imagem em um container utilizando json patch com array posicional"